src/main/java/com/minecolonies/compatibility/tinkers/TinkersWeaponHelper.java

```diff
@@ -0,0 +1,36 @@
+package com.minecolonies.compatibility.tinkers;
+
+import net.minecraft.item.ItemStack;
+import slimeknights.tconstruct.library.tools.SwordCore;
+import slimeknights.tconstruct.library.utils.ToolHelper;
+
+/**
+ * Class to check if certain tinkers items serve as weapons for the guards.
+ */
+public final class TinkersWeaponHelper
```

Comment:
this is the working class. We need a fallback class and the @Optional annotations. This will not work!